src/controller/MahjongRound.ts

```diff
@@ -2,18 +2,21 @@
  * Please read the specs below for the functionality that the prototype should support.
  */
 
-export enum WIND {
+import {calculateHandValue, MANGAN_BASE_POINT} from "./Points";
+
+export enum Wind {
 	EAST,
 	SOUTH,
 }
 
-export enum actionType {
-	RON, // a rons b
-	TSUMO, // a tsumos
-	CHOMBO, // a chombos
-	TENPAI, // a pays tenpai money to b
-	DEAL_IN_PAO, // a deals in b while being pao
-	SELF_DRAW_PAO, // b self-draw; a was pao
+export enum ActionType {
+	RON,
+	TSUMO,
+	CHOMBO,
+	TENPAI,
+	DEAL_IN_PAO,
+	SELF_DRAW_PAO,
+	NAGASHI_MANGAN,
 }
 
 export interface Person {
```

Comment:
In our current project, we model a Person using the Player schema, which has many different properties. I also think we should use the playerId as the key instead of playerName. I would prefer if we kept consistency with the original table fields to make it easier to integrate in